client/app/components/land-use-action.hbs

```diff
@@ -25,10 +25,10 @@
   {{/if}}
 
   {{#each this.sortedSelectedActions as |landUseAction|}}
-    <Ui::Fieldset class="fieldset relative scale-fade-in" as |Q|>
-      <Q.legend data-test-action-name="{{landUseAction.name}}">
+    <Ui::Question class="fieldset relative scale-fade-in" as |Q|>
+      <Q.prompt data-test-action-name="{{landUseAction.name}}">
```

Comment:
I think the convention goes that components should always be capitalized. So this should be 👍 
```
<Q.Prompt />
```

Reply:
Changed.